frontend/components/commons/results/ResultsRecord.vue

```diff
@@ -117,6 +120,9 @@ export default {
       );
       return formattedVectors;
     },
+    recordEventTimestamp() {
+      return this.$moment(this.item.event_timestamp).format("YYYY-MM-DD HH:mm");
+    },
```

Comment:
externalized the format properties as const "YYYY-MM-DD HH:mm" in another file.properties.js